src/test/java/com/trello/boards/NewBoardTest.java

```diff
@@ -11,27 +11,21 @@
 import org.testng.Assert;
 import org.testng.annotations.AfterClass;
 import org.testng.annotations.AfterMethod;
-import org.testng.annotations.BeforeClass;
 import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.Test;
 
 public class NewBoardTest {
 
     private ChromeDriver driver;
```

Comment:
I don't see where you init driver in this class.
Before every PR, please make sure that all tests are passed.